src/test/resources/outputs/partiql/basics/simple.sql

```diff
@@ -111,22 +111,23 @@ NOT ("default"."T1");
 "default"."T1" IS NOT DECIMAL;
 
 --#[expr-24]
-"default"."T1" IS DOUBLE PRECISION;
+"default"."T1" IS FLOAT;
 
 --#[expr-25]
-"default"."T1" IS NOT DOUBLE PRECISION;
```

Comment:
self-review: original test behavior was wrong. Input (https://github.com/partiql/partiql-scribe/blob/vNext/src/test/resources/inputs/basics/simple.sql#L109-L113) was `IS FLOAT / IS NOT FLOAT`, which should map to `FLOAT` not `DOUBLE PRECISION`.